README.md

```diff
@@ -4983,6 +4983,15 @@ The database has to be fully synced. Then, run the tests with:
 elixir --sname aeternity@localhost -S mix test.integration
 ```
 
+### Devmode tests
+
+These tests allow you to create your own transactions using the devmode (plus the JS SDK). To add newer tests you need to:
+
+1. Add the transactions creation on `node_sdk/index.js`.
+2. Run the JavaScript file using `docker-compose -f docker-compose-dev.yml run node_sdk node index.js`.
```

Comment:
Needed `npm i @aeternity/aepp-sdk` under `node_sdk` because the global is not added to the container